PermanentOrg/web-app
# Per 9929 show password strength

This pull request displays the strength of a user's password when registering a new account.

Steps to test: 
1. Go to the register page
2. Start inputting a password
3. A message will appear below with the strength of it
